azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -292,6 +292,12 @@ func resourceMsSqlDatabase() *schema.Resource {
 				},
 			},
 
+			"geo_backup_policy": {
```

Comment:
Could we rename this to 
```suggestion
			"geo_backup_enabled": {
```
to match other boolean properties

Reply:
No problem. Will change in next commit.